app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -143,7 +147,7 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
   }
   // If the user got here by opting-in to in-person proofing, when skipDocAuth === true,
   // then set steps to inPersonSteps
```

Comment:
Should we update this comment? (skipDocAuth === true)

Reply:
yes, thank you!

Reply:
As a heads-up, I still see the `(skipDocAuth === true)` comment here.  Has the most recent version of the code been pushed to github?